lib/atomic_web/controllers/enrollments_controller.ex

```diff
@@ -0,0 +1,29 @@
+defmodule AtomicWeb.EnrollmentController do
+  use AtomicWeb, :controller
+
+  alias Atomic.Activities
+  alias Atomic.Repo
+  alias Atomic.Accounts.User
+  alias Atomic.Activities.Activity
+
+  @doc """
+    Updates an user's enrollment participation for an activity.
+  """
+  def update(conn, %{"activity_id" => activity_id, "user_id" => user_id}) do
+    user = Repo.get!(User, user_id)
+    activity = Repo.get!(Activity, activity_id)
+    enrollment = Activities.get_enrollment(activity, user)
+
+    if enrollment do
+      Activities.update_enrollment(enrollment, %{present: true})
+
+      conn
+      |> put_flash(:info, "Participation confirmed")
+      |> redirect(to: "/activities/#{activity.id}")
```

Comment:
```suggestion
      |> redirect(to: ~p"/activities/#{activity.id}")
```

> Using ~p for route paths ensures our application paths and URLs stay up to date with the router definitions. The compiler will catch bugs for us, and let us know when we change routes that are referenced elsewhere in our application.

Source: https://hexdocs.pm/phoenix/routing.html

Reply:
Shouldn't we use `Routes.activity_show_path/3` here?